### ISSUE TITLE
Cannot dow Arabia for dozens of turns, supposedly due to a peace treaty

### ISSUE THREAD

**Issue by danielsolaro** [LoneGazebo/Community-Patch-DLL#11039]
_1. Mod Version: 4.9.1

_2. DirectX Version_ DX 11

_3. List of Other Mods_
Community Patch
Vox Populi
VP - EUI Compatibility Files
Squads for VP
VP Events Overhaul
Reforestation for VP
Devotion Ideology
Religious Victory
More Unique Components for VP
SMAN's Duchy of Brittany
GH's Catalonia
Change VP Options *
Cambodian Civilization
Colonialist Legacies' - Phillipine Republic for Vox Populi
Colonialist Legacies' Canadian Dominion for VP
Colonialist Legacies' Inuit for VP
Even More Resources for Vox Populi
Favorite Religion Randomizer
Global - Random City Names
Goths for Vox Populi
Historical Religions Edit
Hokath's Afghanistan for VP
InGame Editor+
Jarcast & Hokath's Tibet for VP
League Project Alerts & Assign All Cities
Leugi's Israel for VP
Manchus for Vox Populi
MC and LITE's Nubia for VP
Misc Tweaks for VP *
New Beliefs Mods
Notification Settings
Pineappledan's Chola for Vox Populi
Pineappledan's Phoenician Civilization
Pineappledan's Vox Populi Tweaks *
Quick Turns
Religion - Permanent Pantheons
Tomatekh's Kyivan Rus' (Yaroslav the Wise) for VP
Upgrade All Units Button in Military Overview
WH - Military Log for VP
World Congress Reformation

* Modified by me

_4. Describe the Issue_
NOTE: its the same game from https://github.com/LoneGazebo/Community-Patch-DLL/issues/11033 . I hope its ok to just reuse that mod folder with a new save I attached this time.

Have been unable to declare war on Arabia for a long ass time. They are down to 2 cities and have no allies outside a city state. I have one three allies and one vassal.
I initially thought it had to do with my allies or some old alliance Arabia had, but its been quite a while and this peace treaty never ends. I could force a dow through IGE.
______________________________________________________________________________________
_5. Save Game From 1 Turn Before
contains a save of turn 331 right before I used IGE to force the dow, and contains a bunch of autosaves from previous turns in case you need to check anything
[save on mediafire](https://www.mediafire.com/file/u999q30coqrx3vb/saveforreport.rar/file)

_8. Steps to reproduce the Issue (Optional)_
no idea, sorry

_9. Screenshots of the Issue (Optional)_
![20240622133343_1](https://github.com/LoneGazebo/Community-Patch-DLL/assets/8516353/9d0654af-14c0-4372-a50e-e1f7c57e3372)

**Comment by danielsolaro** [comment]
Damn forgot the victory condition bug, too, on the same game. If you can take a quick look at the victory screen, please, else I will take a screenshot later. Issue was it showing less required capitals to win domination and not counting the greek vassal (they still have their capital).

**Comment by azum4roll** [comment]
Potential duplicate of #11019. Did you sign a third-party peace deal?

**Comment by danielsolaro** [comment]
I honestly don't remember, but I have time now to check stuff. Will load the game and check my past deals.
if you need me to look at anything else its a good time :P

**Comment by danielsolaro** [comment]
No third party peace treaties.

**Comment by saamohod** [comment]
> Potential duplicate of #11019. Did you sign a third-party peace deal?

I'm having the same issue in my game. I've signed a peace treaty brokered by a third party by paying me some gold per turn. Dozens turns later I'm still unable to DoW them, it says I can't declare because I'm having peace treaty, but the peace treaty has expired long ago.

**Comment by axatin** [comment]
I assume this was a third-party deal. As discussed in #11019, expired third-party peace deals don't show up in the deal history (fixed for the next version). @danielsolaro Do you still have earlier save files, and can you use them to check again it is was a third-party deal?

**Comment by danielsolaro** [comment]
@axatin I assume you intended to post a screenshot? I will check if I have the saves and if it loads correctly now

**Comment by axatin** [comment]
In the autosaves a few turns after you made peace, you should be able to find the deal in the deal history, either as a normal peace deal or as a third-party one. Can you check which one it is?

**Comment by danielsolaro** [comment]
I don't know which peace deal you were talking about, that's the thing. Which deal would be "this" that you mentioned. 
So I went back and loaded a few different auto saves for any deal that looked sus.
- I made peace directly with the arabs once on turn 48, ending at 63.
- I made peace with arabs through japanese on turn 107, so the full deal ended at turn 177.
![20240628221602_1](https://github.com/LoneGazebo/Community-Patch-DLL/assets/8516353/d327ca3a-4489-4b4c-a4ac-0295631fe1ff)
We are at turn 331 where I posted this thread.
